src/rpc/blockchain.cpp

```diff
@@ -2247,6 +2247,108 @@ static UniValue getblockfilter(const JSONRPCRequest& request)
     return ret;
 }
 
+/**
+ * Serialize the UTXO set to a file for loading elsewhere.
+ *
+ * @see SnapshotMetadata
+ */
+UniValue dumptxoutset(const JSONRPCRequest& request)
+{
+    RPCHelpMan{
+        "dumptxoutset",
+        "\nWrite the serialized UTXO set to disk.\n"
```

Comment:
In commit "rpc: add dumptxoutset" (b6af8f5443c9891ade4fa5ca92a244e3ff8cac79)

Up to you, but I think documentation could say more about

- Data format 
- What you'd use this function for, or it's mostly meant to be useful for developers
- Status of the implementation. If this is experimental, whether the data format might change, if there's going to be a corresponding load rpc method